lib/ansible/plugins/doc_fragments/eric_eccli.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) 2019 Ericsson AB.
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import (absolute_import, division, print_function) 
+__metaclass__ = type
+
+class ModuleDocFragment(object):
```

Comment:
This file is not required.

Reply:
@ganeshrn 
Maybe this file cannot be delete if adds new document(platform_eric_eccli.rst), since we need to add the path of new document in it. otherwise will not be linked new document.